### PR TITLE
Add a test plan for QUARKUS-2736

### PR DESCRIPTION
**Pull request by michalvavrik**
### Links

JIRA: https://issues.redhat.com/browse/QUARKUS-2736

### Reminder for considerable topics

 - [x] Make sure you have considered the following areas when preparing the test plan:

   - Logging
   - Tracing
   - Metrics
   - Security
   - OpenAPI
   - Data sources
   - Frontend
   - Qute

### CODE REVIEW

**Review by mjurc — APPROVED** [review]
Good plan, thanks.

File: QUARKUS-2736.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+Default exception mappers introduced by RESTEasy Classic are covered upstream as the mappers were actually disabled in
+reaction to failing tests.
+
+Elytron Security migration is sufficiently covered by Quarkus QE Test Suite, Quarkus Quickstarts run as part of upstream CI
+and upstream integration modules.
+
+## Impact on test suites and test environment
+
+Beefy Scenarios should cover newly introduced configuration properties that enforces legacy behavior and provides
```
Comment (rsvoboda) [inline QUARKUS-2736.md:103]: This should be adjusted to Quarkus QE TS, this will not be covered in Beefy TS.

Comment (michalvavrik) [reply QUARKUS-2736.md:103]: np, will do in a separate PR

Comment (michalvavrik) [reply QUARKUS-2736.md:103]: here you go https://github.com/quarkus-qe/quarkus-test-plans/pull/137

**Pull request merged**